spec/requests/users_controller_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe 'users Controller', type: :request do
+  describe 'GET /index' do
+    context 'when the page is loaded' do
+      before do
+        get '/users'
+      end
+      it 'returns a 200 status code' do
+        expect(response).to have_http_status(200)
+      end
+      it 'renders the index template' do
+        expect(response).to render_template('index')
+        expect(response.body).to render_template('index')
+      end
+      it 'includes number of posts' do
+        expect(response.body).to include('Number of posts')
+      end
+    end
+  end
+
+  describe 'GET /show' do
+    context 'when the page is loaded' do
+      before do
+        get '/users/1'
+      end
+      it 'returns a 200 status code' do
+        expect(response).to have_http_status(200)
+      end
+      it 'renders the show template' do
+        expect(response).to render_template('show')
+        expect(response.body).to render_template('show')
+      end
+      it 'includes see all posts' do
+        expect(response.body).to include('See all posts')
+      end
+    end
+  end
```

Comment:
- [ ] Some of these tests failed. Kindly take a look at the attachment and fix the falling tests.
<img width="323" alt="Code_0029c33YUM" src="https://user-images.githubusercontent.com/81468675/179758026-0e440aa3-6fa5-45fd-8e23-7fcfeb5a2e82.png">